.github/ISSUE_TEMPLATE/one_line_request.yml

```diff
@@ -0,0 +1,20 @@
+name: 1-Line art Request
+description: Suggest a new 1-Line art to be added to the package
+title: "[1-Line Art]: "
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Write the 1-Line art you want to add
+      placeholder: >
+        ex: (っ◕‿◕)っ 
+    validations:
+      required: false
```

Comment:
I think this item should be `true`

Reply:
fixed in [7dcf2a6](https://github.com/sepandhaghighi/art/pull/265/commits/7dcf2a62162ca0c223226e970ba03f5bd69c7c85).